src/Compilers/CSharp/Portable/Symbols/Metadata/PE/NullableTypeDecoder.cs

```diff
@@ -16,21 +16,22 @@ internal static class NullableTypeDecoder
         internal static TypeSymbolWithAnnotations TransformType(
             TypeSymbolWithAnnotations metadataType,
             EntityHandle targetSymbolToken,
-            PEModuleSymbol containingModule)
+            PEModuleSymbol containingModule,
+            INonNullTypesContext nonNullTypesContext)
```

Comment:
>INonNullTypesContext nonNullTypesContext) [](start = 12, length = 41)

This seems a significant change/fix, but I didn't see test changes corresponding to it. Do we have adequate coverage?

Reply:
> This seems a significant change/fix, but I didn't see test changes corresponding to it. Do we have adequate coverage?

Why do you see this as a significant change? Context, which was "inferred" from TypeSymbolWithAnnotations before is now passed explicitly. The final goal is to get rid of INonNullTypesContext completely, but we have to do it in steps to keep changes to a reasonable size.

---
In reply to: [230539140](https://github.com/dotnet/roslyn/pull/30913#discussion_r230539140) [](ancestors = 230539140)

Reply:
Thanks for the clarification. Makes sense

---
In reply to: [230957799](https://github.com/dotnet/roslyn/pull/30913#discussion_r230957799) [](ancestors = 230957799,230539140)